pkg/services/ngalert/metrics/scheduler.go

```diff
@@ -91,6 +92,15 @@ func NewSchedulerMetrics(r prometheus.Registerer) *Scheduler {
 			},
 			[]string{"org"},
 		),
+		SimpleNotificationRules: promauto.With(r).NewGaugeVec(
+			prometheus.GaugeOpts{
+				Namespace: Namespace,
+				Subsystem: Subsystem,
+				Name:      "simple_notification_rules",
```

Comment:
NIT: Should we stay in line with the prefix for other rule-oriented metrics?
```suggestion
				Name:      "rule_simplified_routing",
```

Reply:
I used `rule_group_rules` and `schedule_alert_rules` as the template for the name, since this is a gauge of count.